x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/service_overview/instance_details.spec.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { omit } from 'lodash';
+import { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+import { getServiceNodeIds } from './get_service_node_ids';
+
+type ServiceOverviewInstanceDetails =
+  APIReturnType<'GET /internal/apm/services/{serviceName}/service_overview_instances/details/{serviceNodeName}'>;
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const synthtrace = getService('synthtrace');
+
+  const start = '2023-08-22T00:00:00.000Z';
+  const end = '2023-08-22T01:00:00.000Z';
+
+  describe('Service Overview', () => {
+    let client: ApmSynthtraceEsClient;
+
+    before(async () => {
+      client = await synthtrace.createApmSynthtraceEsClient();
+    });
+
+    describe('Instance details', () => {
+      describe('when data is not loaded', () => {
+        it('handles empty state', async () => {
+          const response = await apmApiClient.readUser({
+            endpoint:
+              'GET /internal/apm/services/{serviceName}/service_overview_instances/details/{serviceNodeName}',
+            params: {
+              path: { serviceName: 'opbeans-java', serviceNodeName: 'foo' },
+              query: {
+                start,
+                end,
+              },
+            },
+          });
+
+          expect(response.status).to.be(200);
+          expect(response.body).to.eql({});
+        });
+      });
+
+      // FLAKY: https://github.com/elastic/kibana/issues/177494
```

Comment:
Can you please remove all the comments if the tests are fixed? (Also would be nice to run the flaky test runner if any tests were skipped)